src/controller.cpp

```diff
@@ -609,6 +609,17 @@ void Controller::setState(State state) {
   emit MozillaVPN::instance()->recordGleanEventWithExtraKeys(
       GleanSample::controllerStep,
       {{"state", QVariant::fromValue(state).toString()}});
+
+  // Specific events for on and off state to aid with analysis
+  if (state == StateOn) {
```

Comment:
move these to telemetry.cpp. See https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/src/telemetry.cpp#L43